store/postgres/src/entities.rs

```diff
@@ -1103,7 +1126,12 @@ impl JsonStorage {
             None => return Ok(()),
             Some(event) => event,
         };
-
+        let event_id = match history_event.id {
+            Some(id) => id,
+            None => unreachable!(
+                "we always allocate an event_id when we need to record metadata history"
+            ),
+        };
```

Comment:
This is an `expect`.

Reply:
D'oh. Thanks for pointing that out. Fixed.